src/clr-angular/modal/modal.spec.ts

```diff
@@ -146,6 +148,30 @@ describe('Modal', () => {
     })
   );
 
+  it(
+    'should not emit clrModalOpenChange - animation will do that for us',
+    fakeAsync(() => {
+      /**
+       * Needed just to mock the event so I could enter the `if` statement.
+       */
+      const fakeAnimationEvent: AnimationEvent = {
+        fromState: '',
+        toState: 'void',
+        totalTime: 0,
+        phaseName: '',
+        element: {},
+        triggerName: '',
+        disabled: false,
+      };
+
+      spyOn(getModalInstance(fixture)._openChanged, 'emit');
+      getModalInstance(fixture).close();
+      expect(getModalInstance(fixture)._openChanged.emit).not.toHaveBeenCalled();
+      getModalInstance(fixture).fadeDone(fakeAnimationEvent);
+      expect(getModalInstance(fixture)._openChanged.emit).toHaveBeenCalled();
```

Comment:
Let's change this to `toHaveBeenCalledXTimes(1)` so we can be sure its not duplicated.

Reply:
+1

Reply:
sure, sounds better